src/Batch/FileConventions/Src/AzureBatchFileConventions/Microsoft.Azure.Batch.Conventions.Files.nuget.proj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Batch.Conventions.Files">
-      <PackageVersion>0.9.5-preview</PackageVersion>
+      <PackageVersion>0.9.5-preview2</PackageVersion>
```

Comment:
We should actually take it to 1.0 - Mark S has been badgering for this for a while.